.gitignore

```diff
@@ -0,0 +1,3 @@
+target/
+**/*.rs.bk
```

Comment:
I don't know for sure, I guess it's about backup files that some editors might create. This line is part of the `.gitignore` that `cargo new` creates, so I included it. I can remove it, if you prefer.